portfolio/src/main/webapp/script.js

```diff
@@ -31,3 +31,57 @@ function addRandomFact() {
   const factContainer = document.getElementById('randomFact');
   factContainer.innerText = fact;
 }
+
+/**
+* Retrieves comments when page loads.
+*/
+const getComments = () => {
+  fetch('/comments').then(response => response.json()).then((comments) => {
+    const commentContainerElement = document.getElementById('comment-container');
+    const titleElement = document.createElement("h2");
+
+    //Clean element
+    commentContainerElement.innerHTML = '';
+
+    //If no comments, display an alert, else append comments to the container element.
+    if(comments.length === 0) {
```

Comment:
```suggestion
    if (comments.length === 0) {
```

Reply:
Can comments be null here, or in that case would it be an empty list? As in null.length vs. [].length?